component/Applaud/AllAplaud.jsx

```diff
@@ -77,6 +77,10 @@ function AllAplaud({ user }) {
 
     [filterByUserId, allOrgApplaud]
   );
+  const disabledDate = (current) => {
```

Comment:
Create a date helper file and move all duplicate code related to dates and use them in components